management/dbaas/xtra_db_cluster_test.go

```diff
@@ -0,0 +1,217 @@
+package dbaas
+
+import (
+	"testing"
+
+	dbaasClient "github.com/percona/pmm/api/managementpb/dbaas/json/client"
+	"github.com/percona/pmm/api/managementpb/dbaas/json/client/xtra_db_cluster"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+const (
+	kubernetesClusterName = "api-test-k8s-cluster"
+)
+
+//nolint:funlen
+func TestXtraDBClusterServer(t *testing.T) {
+	if pmmapitests.Kubeconfig == "" {
+		t.Skip("Skip tests of XtraDBClusterServer without kubeconfig")
+	}
+	registerKubernetesCluster(t, kubernetesClusterName, pmmapitests.Kubeconfig)
+
+	t.Run("BasicXtraDBCluster", func(t *testing.T) {
+		paramsFirstPXC := xtra_db_cluster.CreateXtraDBClusterParams{
+			Body: xtra_db_cluster.CreateXtraDBClusterBody{
+				KubernetesClusterName: kubernetesClusterName,
+				Name:                  "first.pxc.test.percona.com",
+				Params: &xtra_db_cluster.CreateXtraDBClusterParamsBodyParams{
+					ClusterSize: 3,
+					Proxysql: &xtra_db_cluster.CreateXtraDBClusterParamsBodyParamsProxysql{
+						ComputeResources: &xtra_db_cluster.CreateXtraDBClusterParamsBodyParamsProxysqlComputeResources{
+							CPUm:        1,
+							MemoryBytes: "64",
+						},
+					},
+					Pxc: &xtra_db_cluster.CreateXtraDBClusterParamsBodyParamsPxc{
+						ComputeResources: &xtra_db_cluster.CreateXtraDBClusterParamsBodyParamsPxcComputeResources{
+							CPUm:        1,
+							MemoryBytes: "64",
+						},
+					},
+				},
+			},
+		}
+
+		paramsFirstPXC.WithContext(pmmapitests.Context)
```

Comment:
Can we set Context while initializing the variable?

Reply:
The same in all places.

Reply:
Yes. Changed.